backend/src/StamAcasa.Common/Services/Emailing/SmtpSender.cs

```diff
@@ -26,13 +27,33 @@ public async Task SendAsync(Email email, CancellationToken cancellationToken)
 
                 await client.AuthenticateAsync(_options.User, _options.Password, cancellationToken);
 
+                var body = new TextPart(TextFormat.Html) { Text = email.Content };
                 var message = new MimeMessage
                 {
                     Sender = new MailboxAddress(email.FromName, email.FromEmail),
                     Subject = email.Subject,
-                    Body = new TextPart(TextFormat.Html) { Text = email.Content },
+                    Body = body,
                     To = { new MailboxAddress(email.To) }
                 };
+                if (email.Attachment != null)
+                {
+                    var attachment = new MimePart("image", "gif")
```

Comment:
Is this the correct Mime type?
Just asking, looks wierd :)

Reply:
No, good catch. I used some images first for testing and I forgot this. I'll update it!